src/realm/impl/transact_log.hpp

```diff
@@ -81,6 +81,8 @@ enum Instruction {
     instr_LinkListNullify = 36, // Remove an entry from a link list due to linked row being erased
     instr_LinkListClear = 37,   // Ramove all entries from a link list
     instr_LinkListSetAll = 38,  // Assign to link list entry
+    instr_AddSubtableSearchIndex = 39,  // Add search index to subtable
+    instr_RemoveSubtableSearchIndex = 40   // Remove search index from subtable
```

Comment:
I think it should be possible to reuse the existing `instr_[Add|Remove]SearchIndex` instructions. It would be nice to do that so we don't add compounding complexity to sync who would need to handle these new instructions too. I think we at least don't need to know the table index because the replicator can [select subtables](https://github.com/realm/realm-core/blob/master/src/realm/impl/transact_log.hpp#L958) right?